### ISSUE TITLE
Technique G195 is Advisory not Sufficient for 2.4.7: Focus Visible (?!)

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2468]
Technique G195 is Advisory not Sufficient for 2.4.7: Focus Visible (?!)
https://w3c.github.io/wcag/techniques/general/G195

I see G195 is Sufficient for 2.4.7, but the examples given do not make a FAIl turn into a PASS but instead just 'increase' visibility.
Especially now we have [Success Criterion 2.4.11: Focus Appearance (Minimum)](https://w3c.github.io/wcag/understanding/focus-appearance-minimum) it is clear that making something 'more visible' is NOT 2.4.7 but a Sufficient technique for 2.4.11

So this Techniques should be changed to an Advisory technique (and of course advisory techniques can be sufficient too!)

**Comment by cstrobbe** [comment]
I don't understand why technnique G195 shouldn't be marked as sufficient. If this is because of the examples, what are the specific criteria by which the examples don't pass? (Do they need to be replaced?) If it is because something in the description, which specific part of the description makes the technique advisory?

Making the focus more visible than the browser's default focus is still highly desirable. In Firefox, for example, the default focus indicator is simply much too thin, especially on links and other components that use white (or light) text on a darker background and on image links.

Some changes may improve the technique, however:

1. "by creating a visible one" -> "by creating a highly visible one".
2. "...  a thick line, and other visual indicators ..." -> "...  a thick line, or other visual indicators ... (or a combination of these)" ?
3. Test procedure: Possibly add the following note: "For the purpose of the test procedure, it does not matter whether the "border" is implemented using [CSS border properties](https://www.w3.org/TR/css-backgrounds-3/#borders) such as `border-color`, `border-width` and `border-style`, or [outline properties](https://www.w3.org/TR/css-ui-3/#outline-props)."
4. The test procedure focuses exclusive on border/outline, whereas changes on background colour or colour inversion may also work. However, we already have [C15](https://www.w3.org/WAI/WCAG21/Techniques/css/C15.html) and a placeholder for "Using a change of contrast ratio to provide a visible focus indicator" as alternative techniques.
5. Either replace example 3 with one that relies on a change in the border or outline, or expand the test procedure so it takes background colour changes into account.

Purely editorial change: "The objective (...) is enhance" -> "The objective (...) is to enhance".
